ccm_web/client/src/utils/fileUtils.ts

```diff
@@ -5,14 +5,18 @@ const prepDownloadDataString = (data: string): string => {
 }
 
 /*
-Takes a file name and replaces the string segment without a period
+Takes a file name and replaces the string segment(as optional) without a period
 just before the file name extension with that segment plus a new provided ending.
+the default string segment is '-formatted'
 Examples:
-  createOutputFileName('old-file-name.csv', '-puff') returns 'old-file-name-puff.csv'
-  createOutputFileName('old-file-name', '-puff') returns 'old-file-name-puff
-  createOutputFileName('old.file.name.csv', '-puff') returns 'old.file.name-puff.csv'
+  createOutputFileName('old-file-name.csv', '-new') returns 'old-file-name-new.csv'
+  createOutputFileName('old-file-name') returns 'old-file-name-formatted
+  createOutputFileName('old.file.name.csv', '-alt') returns 'old.file.name-alt.csv'
 */
-function createOutputFileName (oldFileName: string, newfileEnding: string): string {
+function createOutputFileName (oldFileName: string, newfileEnding?: string): string {
```

Comment:
I made this optional so we can have a choice for a future feature download as needed